package.json

```diff
@@ -76,16 +76,16 @@
     "coveralls": "3.1.1",
     "enhanced-resolve": "4.5.0",
     "envalid": "7.2.2",
-    "eslint": "7.32.0",
+    "eslint": "8.4.1",
     "eslint-config-prettier": "8.3.0",
-    "eslint-loader": "4.0.2",
```

Comment:
This [loader](https://www.npmjs.com/package/eslint-loader) has been deprecated and replaced with `eslint-webpack-plugin`